partials/examples/categories.html

```diff
@@ -0,0 +1,6 @@
+<p>
+    This example shows default renderer for categories.
```

Comment:
Explain what Categories actually are and how they relate to the rendered from (Tabs).